showcase/src/app/demo-ui/pages/table/demo-table-page.component.html

```diff
@@ -73,6 +73,15 @@ <h1 translate>SHOWCASE.DEMO.SHARED.EXAMPLE_VIEWER_LIST</h1>
 		<showcase-table-with-custom-styling></showcase-table-with-custom-styling>
 	</example-viewer>
 
+	<example-viewer
+		id="collapse"
+		exampleTitle="SHOWCASE.DEMO.TABLE.WITH_CUSTOM_CELL_RENDERING"
+		filesPath="table/with-custom-cell-rendering/table"
+		[extensions]="['HTML', 'TS', 'SCSS']"
```

Comment:
Since you don't provide any css, could you please remove the `'SCSS'` mention? 😊

Reply:
done